src/index.ts

```diff
@@ -145,13 +112,22 @@ interface QuoteCSS {
   caption: string;
 }
 
+/**
+ * @typedef {Enum} Alignment
+ * @description Enum for Quote Alignment
+ */
+enum Alignment {
+  left = "left",
```

Comment:
its better to [capitalize](https://www.typescriptlang.org/docs/handbook/enums.html) enum constants

Reply:
Done 👍🏼

Reply:
I mean `Alignment.Left` and `Alignment.Right` like in the official linked above